packages/electron-test-helpers/src/client.ts

```diff
@@ -11,16 +11,16 @@ interface ClientTestHelpers {
 export function makeClientForPlugin ({
   config = {},
   schema = {},
-  plugin = undefined
-}: { config?: object, schema?: object, plugin?: Plugin } = {}): ClientTestHelpers {
+  plugins = undefined
```

Comment:
could we just default this to an empty array and remove the logic on line 23?

Reply:
yes we could! updated 👍🏾